addons/stock_account/__openerp__.py

```diff
@@ -27,7 +27,10 @@
     'description': """
 WMS Accounting module
 ======================
-This module makes the link between the 'stock' and 'account' modules and allows you to create accounting entries to value your stock movements
+This module makes the link between the 'stock' and 'account' modules and allows
```

Comment:
Although this completion is good, please don't modify this. OCB and Odoo patches are intended to be minimal.